src/Microsoft.Identity.Client/Platforms/Android/UIParent.cs

```diff
@@ -49,7 +49,8 @@ public sealed class UIParent
         /// <summary>
         /// Default constructor. Should not be used on Android.
         /// </summary>
-        [Obsolete("This constructor should not be used because this object requires a parameters of type Activity. ")]
+        [Obsolete("See https://aka.ms/msal-net-3-breaking-changes", true)]
```

Comment:
>Obsolete [](start = 9, length = 8)

assuming all the obsolete stuff moved to the migrationaid subfolder (may require #ifdef and marking the class partial)

Reply:
Not for UIParent.  I'm marking it obsolete inline since it's platform-specific and there are multiple implementations of the same class.  so i'm just updating the obsolete documentation for these and throwing exception like we're doing for the other stuff that is moving to migrationaid.cs

---
In reply to: [268374935](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1002#discussion_r268374935) [](ancestors = 268374935)